content/en/docs/install.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Installing Tekton Pipelines"
+linkTitle: "Install"
+weight: 20
+menu:
+  main:
+    weight: 20
+---
+
+To add Teketon to the Kubernetes Cluster on version 1.11 or later grant cluster-admin permissions to the current user.
+
+* [Installing Tekton Pipelines](#installing-tekton-pipelines)
+* [Installing Tekton PIpelines on OpenShift/MiniShift](#installing-tekton-pipelines-on-openshiftminishift)
+* [Versions](#versions)
+
+
+## Installing Tekton Pipelines
+
+To add the Tekton Pipelines component to an existing cluster:
+
+1. Run the
+   [`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply)
+   command to install [Tekton Pipelines](https://github.com/tektoncd/pipeline)
+   and its dependencies:
+
+   ```bash
+   kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+   ```
+
+   _(Previous versions will be available at `previous/$VERSION_NUMBER`, e.g.
+   https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.2.0/release.yaml.)_
+
+1. Run the
+   [`kubectl get`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#get)
+   command to monitor the Tekton Pipelines components until all of the
+   components show a `STATUS` of `Running`:
+
+   ```bash
+   kubectl get pods --namespace tekton-pipelines
+   ```
+
+   Tip: Instead of running the `kubectl get` command multiple times, you can
+   append the `--watch` flag to view the component's status updates in real
+   time. Use CTRL + C to exit watch mode.
+
+You are now ready to create and run Tekton Pipelines:
+
+- See [Tekton Pipeline tutorial](./tutorial.md) to get started.
+- Look at the
+  [examples](https://github.com/tektoncd/pipeline/tree/master/examples)
+
+### Installing Tekton Pipelines on OpenShift/MiniShift
```

Comment:
I think we should also talk about the Operator in the case of OpenShift :angel: 
`Minishift` is running OpenShift 3.x which is theoretically too old to support Tekton, so I wonder if we should document it :sweat_smile:

Reply:
Yeah, I was playing around with openshift 4 (CRC). Ill change it to be more relevant.